src/data/modifiers.txt

```diff
@@ -155,8 +155,8 @@ Item	crappy Mer-kin mask	Stench Resistance: +3, Item Drop: -25, Initiative: -50,
 Item	Crimbo hat	Spooky Resistance: +1, Familiar Effect: "1.75xVolley, 1.75xLep, cap 25"
 Item	Crown of the Goblin King	Muscle: +10, Familiar Effect: "2xBarrr, 1xGhuol, cap 17"
 # Crown of Thrones: Put a Familiar In It!
-# Crown of Thrones: That Familiar gains 1 Experience per Fight
 Item	Crown of Thrones	Softcore Only
+# Crown of Thrones: That Familiar gains 1 Experience per Fight
```

Comment:
comments are better but it now goes comment -> item -> other comments, which is probably not what you meant